N24DCCN173/buoi4/bai3.cpp

```diff
@@ -0,0 +1,17 @@
+#include <bits/stdc++.h>
+using namespace std;
+int main()
+{
+    int n; cin >> n;
+    for(int i = 0; i <= n/2; i++)
```

Comment:
for thì hơi thừa
bài này 1 if là ra, em đoán thử cách đó là gì (trả lời ở đây cũng đc khỏi sửa code)

Reply:
int n; cin >> n;
    int a = int(sqrt(n));
    if(sqrt(n) == a) cout << "exactly";
    else cout << "error";
vậy đúng không anh

Reply:
> int n; cin >> n; int a = int(sqrt(n)); if(sqrt(n) == a) cout << "exactly"; else cout << "error"; vậy đúng không anh

đúng rồi, e thấy thay vì lặp lại nhiều lần (nếu n = 1000 thì sẽ chạy 500 dòng if) thì chỉ cần chạy 1 if là xong.
Nhớ cập nhật lại mấy bài kia nữa nha

Reply:
ok anh